hail/Makefile

```diff
@@ -109,7 +109,7 @@ cluster_name := cluster-$(shell whoami)-$(shell echo $$RANDOM)
 test-dataproc: install-hailctl-wheel
 	hailctl dataproc start $(cluster_name) --max-idle 10m --vep GRCh37
 	for FILE in `ls python/cluster-tests`; do \
-	  hailctl dataproc submit $(cluster_name) python/cluster-tests/$$FILE || break; done || exit
+	  hailctl dataproc submit $(cluster_name) python/cluster-tests/$$FILE || break -1; done || exit
```

Comment:
While this fails, I don't think it's correct still; I think the break needs to be `exit 1`. Making the other exit command  unnecessary. 

```make
.PHONY: t1 t2 all
all: t1 t2

t1:
	for i in 1 2; do \
		false || break -1; \
		echo $$i; \
	done

t2:
	for i in 1 2; do \
		false || exit 1; \
		echo $$i; \
	done
```
For this makefile, `make t1` outputs:
```
for i in 1 2; do \
	false || break -1; \
	echo $i; \
done
/bin/sh: line 1: break: -1: loop count out of range
make: *** [Makefile:7: t1] Error 1
```
Whereas, `make t2` outputs:
```
for i in 1 2; do \
	false || exit 1; \
	echo $i; \
done
make: *** [Makefile:13: t2] Error 1
```

The first one errors due to a shell syntax issue, the second due to the shell script failing. In both of these, nothing is output.

Reply:
oh, cool. thanks